solution-base/mongodb/Makefile

```diff
@@ -25,7 +25,12 @@ fetch-mongodb-sharded:
 		--version ${CHART_MONGO_SHARDED_VERSION} \
 		--untar \
 		--untardir ${CHART_DIR}
-
 patch:
-	@git apply --check ${PATCH_FILES}
-	@git apply ${PATCH_FILES}
+	@for patch_file in $(PATCH_FILES); do \
+		if [ "$$patch_file" != "${PATCH_DIR}/create-app-creds.patch" ] && [ "$$patch_file" != "${PATCH_DIR}/credentials-from-secret.patch" ]; then \
```

Comment:
why do we exclude these patches?
- if we exclude them, we might as well remove the patches altogether
- instead of re-doing what `git apply` does, would be simpler to to just filter the files in `PATCH_FILES`, something like:
   ```
   PATCH_FILES := $(shell ls -d ${PATCH_DIR}/* | tr '\n' ' ' | grep -v -e create-app-creds.patch -e credentials-from-secret.patch)
   ```

Reply:
as the files were removed , the if was removed as well

Reply:
why not apply as before?
both chunks can be removed....